config-svc/pkg/api/routes_v1.go

```diff
@@ -37,46 +37,35 @@ func (s *Server) PostClustersAdd(ctx echo.Context) error {
 		return err
 	}
 
-	secure := false
-	if data.CouchbaseConfig.Secure != nil && *data.CouchbaseConfig.Secure {
-		secure = true
+	scheme := "http"
+	var useTLS bool
```

Comment:
it might be clearer just to say `useTLS = false`, you do that for the `scheme` variable just prior anyway